src/components/Board.js

```diff
@@ -4,7 +4,7 @@ import Square from "./Square";
 export default () => {
   let result = [];
   for (let i = 1; i <= 9; i++) {
-    result.push(<Square key={i} currentPlayer={"X"} />);
+    result.push(<Square key={i} currentPlayer={"X"} gameOver={false} />);
```

Comment:
I think the `<Square />` component should default to have inner state to have `gameOver = false`. Review my comment in that file